airbyte-integrations/connectors/source-convex/source_convex/source.py

```diff
@@ -0,0 +1,141 @@
+#
+# Copyright (c) 2022 Airbyte, Inc., all rights reserved.
+#
+
+
+from typing import Any, Iterable, List, Mapping, MutableMapping, Optional, Tuple
```

Comment:
a lot of this PR was generated. source_convex/source.py is the main implementation. everything else is configs or tests.

Reply:
> Se generó mucho de este PR. source_convex/source.py es la implementación principal. todo lo demás son configuraciones o pruebas.

R+(10)select+1
#1  #[phone](url)